smriprep/workflows/anatomical.py

```diff
@@ -219,12 +227,33 @@ def init_anat_preproc_wf(
                             run_without_submitting=True)
 
     # 2. Brain-extraction and INU (bias field) correction.
-    brain_extraction_wf = init_brain_extraction_wf(
-        in_template=skull_strip_template[0],
-        template_spec=skull_strip_template[1],
-        atropos_use_random_seed=not skull_strip_fixed_seed,
-        omp_nthreads=omp_nthreads,
-        normalization_quality='precise' if not debug else 'testing')
+    def _is_skull_stripped(imgs):
+        """ Checks that if T1w images are skulled stripped by interrogating
+        extreme planes for all-zeros
+        """
+        def _check_img(img):
+            data = img.get_data()
```

Comment:
```suggestion
            data = np.abs(nb.load(img).get_fdata(dtype=np.float32))
```
Chris' recommendation plus weird in-scanner denoising + resampling that may introduce negative values.